kinetic/distribution.yaml

```diff
@@ -6007,6 +6007,21 @@ repositories:
       url: https://github.com/IntelRealSense/librealsense.git
       version: legacy
     status: maintained
+  librealsense2:
+    doc:
+      type: git
+      url: https://github.com/IntelRealSense/librealsense.git
+      version: tag
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/IntelRealSense/librealsense2-release.git
+      version: 2.23.0-1
+    source:
+      type: git
+      url: https://github.com/IntelRealSense/librealsense.git
+      version: tag
```

Comment:
This [must be a branch name](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#continuous-integration-indexing) on the source repo. Did you mean `master` instead of `tag`?